src/storage/src/client.rs

```diff
@@ -620,10 +683,27 @@ mod v1 {
             assert_eq!(got.custom_time, object.custom_time);
             assert_eq!(got.soft_delete_time, object.soft_delete_time);
             assert_eq!(got.hard_delete_time, object.hard_delete_time);
+            if let Some(object) = &object.owner {
```

Comment:
Like below, these Some/None cases can be more readable with a `match (object.owner, got.owner)`...